third_party/python/BUILD

```diff
@@ -59,15 +62,12 @@ python_wheel(
 python_wheel(
     name = "xmlrunner",
     package_name = "unittest_xml_reporting",
-    hashes = ["530e703bf8f90a867ce42a524fc4cc99c50c0a1522ce7bd010a783698bd4cac9"],
     version = "1.11.0",
     deps = [":six"],
 )
 
-python_wheel(
+pip_library(
```

Comment:
Is there a reason you want to use `pip_library()` here instead of `python_wheel()`? `python_wheel()` is generally more reliable.

Reply:
I think this was because the wheel resolver tool depends on a few things including `six` so they can no longer be python_wheels otherwise we get cyclic dependencies.